docs/contribute/cumulative-docs/example-scenarios.md

```diff
@@ -0,0 +1,537 @@
+:alt:
+:::
+::::
+::::{tab-item} Code
+````
+---
+applies_to:
+  stack: ga
+  serverless: ga
+---
+
+# Spaces
+
+[...]
+
+## Configure a space-level landing page [space-landing-page]
+
+```{applies_to}
+stack: ga
+serverless: unavailable
+```
+````
+::::
+:::::
+
+:::{tip}
+Likewise, when the difference is specific to just one paragraph or list item, the same rules apply.
+Just the syntax slightly differs so that it stays inline: `` {applies_to}`serverless: unavailable` ``.
+:::
+
+## Functionality is added to an unversioned product [unversioned-added]
+
+When functionality is _first added_ to an unversioned product/deployment mode,
+how it is labeled depends on if the functionality is in technical preview, beta, or GA.
+
+### If the section lifecycle is the same as the page level [unversioned-added-same]
+
+% FOR THE REVIEWER: IS THERE AN EXAMPLE?
+% I actually couldn't find an example where unversioned content was added
+% without also adding content for a versioned product.
+For example, on the [<page>](#) page we added content about new <something> functionality that was added to Serverless in GA.
+Since the page's frontmatter already includes `serverless: ga`, there is no need to label the added content.
+
+:::::{tab-set}
+::::{tab-item} Image
+% :::{image} ./images/
+% :screenshot:
+% :alt:
+% :::
+::::
+::::{tab-item} Code
+::::
+:::::
+
+% FOR THE REVIEWER: IS THIS TRUE?
+::::::{important}
+However, if the functionality is also applicable to a specific version of a versioned product/deployment mode,
+label the content with both versioned and unversioned applicability information.
+
+For example, on the [Lens](https://www.elastic.co/docs/explore-analyze/visualize/lens) page we added information
+about a new option that was added to Serverless in GA and the Elastic Stack in GA in 9.1.0.
+Even though it is added to Serverless, an unversioned product, in the same lifecycle state as the page-level annotation,
+we still include an inline annotation to make it clear that this is not only available in the Elastic Stack.
+
+:::::{tab-set}
+::::{tab-item} Image
+:::{image} ./images/example-unversioned-added-same-exception.png
+:screenshot:
+:alt:
+:::
+::::
+::::{tab-item} Code
+```
+---
+applies_to:
+  stack: ga
+  serverless: ga
+---
+
+# Lens [lens]
+
+[...]
+
+#### Tables
+
+**Density** {applies_to}`stack: ga 9.1` {applies_to}`serverless: ga`
+:   Make the table more or less compact. Choose between **Compact**, **Normal** (default), and **Expanded**.
+```
+::::
+:::::
+::::::
+
+### If the section lifecycle is different than the page level
+
+% FOR THE REVIEWER: IS THERE AN EXAMPLE?
+% I actually couldn't find an example of this scenario.
+For example, on the [<page>](#) page we added content about new <something> functionality that was added to Serverless in preview.
+Since this is different than the page-level applicability in the frontmatter, `serverless: ga`,
+label the content about the new functionality with `serverless: preview`.
```

Comment:
```suggestion
For example, on the [Dashboard controls](https://www.elastic.co/docs/explore-analyze/dashboards/add-controls#add-esql-control) page we added content about new ES|QL controls functionality that was added to Serverless in preview.
Since this is different than the page-level applicability in the frontmatter, `serverless: ga`,
label the content about the new functionality with `serverless: preview`.
```
This is also about the stack in addition to serverless but I guess it still demonstrates the point